cmd/README.md

```diff
@@ -0,0 +1,719 @@
+Let's change `chainId`, `gas`, and `gasPrice`. Make sure the `chainId` matches the one in the genesis file. 
+
+``` 
+networks: {
+    local: {
+      //"http://{ip}:{port}/ext/bc/{chainID}/rpc
+      // modify this in the local_rpc.json
+      url: localRPC,
+      chainId: 99999,
+      accounts: [
+        "0x56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027",
+        "0x7b4198529994b0dc604278c99d153cfd069d594753d471171a1d102a10438e07",
+        "0x15614556be13730e9e8d6eacc1603143e7b96987429df8726384c2ec4502ef6e",
+        "0x31b571bf6894a248831ff937bb49f7754509fe93bbd2517c9c73c4144c0e97dc",
+        "0x6934bef917e01692b789da754a0eae31a8536eb465e7bff752ea291dad88c675",
+        "0xe700bdbdbc279b808b1ec45f8c2370e4616d3a02c336e68d85d4668e08f53cff",
+        "0xbbc2865b76ba28016bc2255c7504d000e046ae01934b04c694592a6276988630",
+        "0xcdbfd34f687ced8c6968854f8a99ae47712c4f4183b78dcc4a903d1bfe8cbf60",
+        "0x86f78c5416151fe3546dece84fda4b4b1e36089f2dbc48496faf3a950f16157c",
+        "0x750839e9dbbd2a0910efe40f50b2f3b2f2f59f5580bb4b83bd8c1201cf9a010a"
+      ],
+      gasPrice: 25000000000,
+      gas: 10000000,
+    }
+  }
+```
+
+We also need to make sure `localRPC` points to the right value.
+
+Let's copy `local_rpc.example.json`. 
+
+``` cp local_rpc.example.json local_rpc.json ``` 
+
+Now in `local_rpc.json` we can modify the rpc url to the one we just created. It should look something like this. 
+
+```
+{
+  "rpc": "http://127.0.0.1:9656/ext/bc/2jDWMrF9yKK8gZfJaaaSfACKeMasiNgHmuZip5mWxUfhKaYoEU/rpc"
+}
+```
+
+Now if we go to `./contract-examples`, we can finally run our tests. 
+
+``` npx hardhat test --network local ```
+
+Great they passed! All the functions implemented in the precompile work as expected!
+
+## Step 8: Create Genesis
+
+We can move our genesis file we created in the last step to  `tests/e2e/genesis/`.
+
+```cp /tmp/subnet-evm-genesis.json  tests/e2e/genesis/hello_world.json``` 
+
+## Step 9: Add E2E tests
+
+In `tests/e2e/solidity/suites.go` we can now write our first e2e test!
+It's another nice copy and paste situation. 
+
+``` go
+ginkgo.It("hello world", func() {
+		err := startSubnet("./tests/e2e/genesis/hello_world.json")
+		gomega.Expect(err).Should(gomega.BeNil())
+		running := runner.IsRunnerUp()
+		gomega.Expect(running).Should(gomega.BeTrue())
+		runHardhatTests("./test/ExampleHelloWorld.ts")
+		stopSubnet()
+		running = runner.IsRunnerUp()
+		gomega.Expect(running).Should(gomega.BeFalse())
+	})
+
+	// ADD YOUR PRECOMPILE HERE
+	/*
+			ginkgo.It("your precompile", func() {
+			err := startSubnet("./tests/e2e/genesis/{your_precompile}.json")
+			gomega.Expect(err).Should(gomega.BeNil())
+			running := runner.IsRunnerUp()
+			gomega.Expect(running).Should(gomega.BeTrue())
+			runHardhatTests("./test/Example{YourPrecompile}Test.ts")
+			stopSubnet()
+			running = runner.IsRunnerUp()
+			gomega.Expect(running).Should(gomega.BeFalse())
+		})
+	*/
+```
+
+## Step 10: Run E2E Test
```

Comment:
if possible can we add a step to show how to call this ExampleHelloWorld contract from Remix and show results?